testing/snooze/APKBUILD

```
@@ -0,0 +1,24 @@
+# Maintainer: Leo <thinkabit.ukim@gmail.com>
+pkgname=snooze
+pkgver=0.3
+pkgrel=0
+pkgdesc="Run a command at a particular time"
+url="https://github.com/leahneukirchen/snooze"
+options="!check" # Provides no testsuite
+arch="all"
+license="CC0"
```

The [SPDX][] designation is:

```suggestion
license="CC0-1.0"
```
[SPDX]:https://spdx.org/licenses/

---

Applied, thank you.